src/main/java/com/softserve/academy/spaced/repetition/controller/ExceptionHandlerController.java

```diff
@@ -22,52 +25,47 @@
 import java.net.UnknownHostException;
 import java.util.EnumMap;
 import java.util.List;
+import java.util.Locale;
 import java.util.NoSuchElementException;
 import java.util.stream.Collectors;
 
 @ControllerAdvice
 public class ExceptionHandlerController extends ResponseEntityExceptionHandler {
 
+    @Autowired
+    private MessageSource messageSource;
+    private final Locale locale = LocaleContextHolder.getLocale();
+
     private static final EnumMap<AccountStatus,
             ResponseEntity<MessageDTO>> USER_STATUS_ERROR_RESPONSE = new EnumMap<>(AccountStatus.class);
 
-    static {
-        USER_STATUS_ERROR_RESPONSE.put(AccountStatus.DELETED,
-                new ResponseEntity<>(new MessageDTO("Account with this email is deleted"),
-                        HttpStatus.LOCKED));
-        USER_STATUS_ERROR_RESPONSE.put(AccountStatus.BLOCKED,
-                new ResponseEntity<>(new MessageDTO("Account with this email is blocked"),
-                        HttpStatus.FORBIDDEN));
-    }
```

Comment:
Why did you delete this code? 
It uses in the method `handleUserStatusException`.